integration_tests/src/main/python/spark_session.py

```diff
@@ -48,6 +48,8 @@ def _set_all_confs(conf):
 def reset_spark_session_conf():
     """Reset all of the configs for a given spark session."""
     _set_all_confs(_orig_conf)
+    """We should clear the cache"""
```

Comment:
I think we need this because we are using the same spark session to run on CPU and GPU with the exception of just changing a conf. I could be wrong

Reply:
I would prefer for us to use a comment instead of a doc string.  doc strings are supposed to be the first thing in a function definition.  I think a comment would fit better here.